offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/dataresources/offers/OfferDbConnector.groovy

```diff
@@ -276,6 +281,7 @@ class OfferDbConnector implements CreateOfferDataSource, FetchOfferDataSource, P
                 def items = productGateway.getItemsForOffer(offerPrimaryId)
                 def checksum = resultSet.getString("checksum")
                 def associatedProject = resultSet.getString("associatedProject")
+                def experimentalDesign = resultSet.getString("experimentalDesign")
```

Comment:
Nit: dynamic typing is evil 👿 
Being more explicit is not really much effort...
```suggestion
                String experimentalDesign = resultSet.getString("experimentalDesign")
```

Reply:
Then I would argue to make it explicit for all variables explicit. I just went with the style and thought its ok for variables within a method

Reply:
Do you want me to make it explicit for all of them?

Reply:
I personally would greatly appreciate that 😉

Reply:
will do 👍🏼